docs/experiments-msmarco-passage.md

```diff
@@ -463,3 +463,4 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@BeginningGradeMaker](https://github.com/BeginningGradeMaker) on 2024-01-15 (commit [`85fad8d`](https://github.com/castorini/anserini/commit/85fad8d9a3aa797b12362894e40aa00041bf3664))
 + Results reproduced by [@ia03](https://github.com/ia03) on 2024-01-17 (commit [`85fad8d`](https://github.com/castorini/anserini/commit/85fad8d9a3aa797b12362894e40aa00041bf3664))
 + Results reproduced by [@AlexStan0](https://github.com/AlexStan0) on 2024-01-19 (commit [`edd47ad`](https://github.com/castorini/anserini/commit/edd47ad3b8d833fcf834bb50e9735f35f5d9b79b))
++ Results reproduced by [@tian0106](https://github.com/tian0106) on 2024-01-19 (commit [`c919196`](https://github.com/tian0106/anserini/commit/c9191969d761c4b13ea2549fafc7be3670f5a7a6))
```

Comment:
The commit id should refer to a commit in trunk, i.e., being with https://github.com/castorini/